pkg/cli/vex.go

```diff
@@ -28,22 +70,19 @@ func VEX() *cobra.Command {
 				AuthorRole: role,
 			}
 
-			doc, err := vex.FromPackageConfiguration(buildCfg, vexCfg)
+			doc, err := vex.FromPackageConfiguration(vexCfg, confs...)
 			if err != nil {
-				return err
+				return fmt.Errorf("creating VEX document: %w", err)
 			}
 
-			err = doc.ToJSON(os.Stdout)
-			if err != nil {
-				return err
+			if err := doc.ToJSON(os.Stdout); err != nil {
+				return fmt.Errorf("marshaling VEX document")
```

Comment:
Should we add `...: %w` to show what the error was?

Reply:
Done!